server/fleet/apple_mdm.go

```diff
@@ -526,10 +526,11 @@ type SCEPIdentityCertificate struct {
 // SCEPIdentityAssociation represents an association between an identity
 // certificate an a specific host.
 type SCEPIdentityAssociation struct {
-	HostUUID         string `db:"host_uuid"`
-	SHA256           string `db:"sha256"`
-	EnrollReference  string `db:"enroll_reference"`
-	RenewCommandUUID string `db:"renew_command_uuid"`
+	HostUUID              string `db:"host_uuid"`
+	SHA256                string `db:"sha256"`
+	EnrollReference       string `db:"enroll_reference"`
+	RenewCommandUUID      string `db:"renew_command_uuid"`
+	EnrolledFromMigration bool   `db:"enrolled_from_migration"`
```

Comment:
Does this apply to only the silent migration or does it also apply to the Fleet Desktop migration flow? If not applicable to both, can we rename/add comments for future reference?

Reply:
good idea, added!